commercial/app/views/hosted/guardianHostedArticle.scala.html

```diff
@@ -1,10 +1,211 @@
+                                        title="pinterest">
+                                            <span class='inline-icon__fallback button share-modal__item share-modal__item--pinterest'>Share on Pinterest</span>
+                                            @fragments.inlineSvg("share-pinterest", "icon", List("rounded-icon social-icon social-icon--pinterest"))
+                                        </a>
+                                    </li>
+                                </ul>
+                            </div><!--End of social media icons !-->
+                        </div>
 
-    TODO ...
+                        <div class="content__article-body from-content-api js-article__body">
+                            <p class="intro">
+                                Zootropolis is a city like no other. But when optimistic Judy Hopps arrives, she discovers that being a bunny on a police force of big, tough animals isn’t so easy.
+                            </p>
+                            <p>
+                                <span class="drop-cap hosted-tone-bg--zootropolis">D</span> Determined to prove herself, she jumps at the opportunity to crack a case, even if it means partnering with a fast-talking fox, Nick Wilde. Walt Disney Animation Studios’ “Zootropolis,” is a comedy-adventure directed by Tangled’s Byron Howard.
+                            </p>
 
+                            <!--Beginning of the Con Artist - fox !-->
+                            <div class="animal con-artist">
+                                <p class="animal__stripe-odd">
+                                    <img src="@{page.conArtistPic}" class="animal__stripe-character" alt="@{page.campaign.owner}">
+                                </p>
+                                <div class="animal__info">
+                                    <div class="animal__info-poster">
+                                        <img src="@{page.conArtistPoster}" alt="@{page.campaign.owner}">
+                                    </div>
+                                    <div class="animal__info-about">
+                                        <h3 class="animal__info-about-header hosted-tone--zootropolis">The Con Artist</h3>
+                                        <span>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut eget velit efficitur, viverra velit vel, vehicula sapien. Duis eu ligula augue. Donec sed ligula at massa bibendum sodales. Aenean quis lorem auctor, semper sem at, interdum ligula. Mauris lacinia pharetra augue ac gravida.</span>
+                                        <a href="#" class="animal__info-about-download">Download character poster</a>
+                                    </div>
+                                </div>
+                            </div>
+
+                            <!--Beginning of the Rookie - bunny !-->
+                            <div class="animal rookie">
```

Comment:
That is also a fair point, although I think we should discriminate between [visual and layout patterns](http://danielmall.com/articles/content-display-patterns/).